generators/csharp/sdk/src/endpoint/EndpointGenerator.ts

```diff
@@ -38,6 +38,7 @@ export class EndpointGenerator {
         endpoint: HttpEndpoint;
         rawClientReference: string;
     }): csharp.Method {
+        console.log(`Generating endpoint id ${endpoint.id}`);
```

Comment:
nit: should this use the generator exec context logger? or should it be removed?

Reply:
should be removed, my b